src/Ryujinx.Horizon/Pctl/Ipc/PctlService.cs

```diff
@@ -0,0 +1,1259 @@
+            {
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1004)]
+        public Result ConfirmSnsPostPermission()
+        {
+            if (HorizonStatic.Options.IgnoreMissingServices)
+            {
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1005)]
+        public Result ConfirmSystemSettingsPermission()
+        {
+            if (HorizonStatic.Options.IgnoreMissingServices)
+            {
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1006)]
+        public Result IsRestrictionTemporaryUnlocked(out bool arg0)
+        {
+            if (HorizonStatic.Options.IgnoreMissingServices)
+            {
+                arg0 = default;
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1007)]
+        public Result RevertRestrictionTemporaryUnlocked()
+        {
+            if (HorizonStatic.Options.IgnoreMissingServices)
+            {
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1008)]
+        public Result EnterRestrictedSystemSettings()
+        {
+            if (HorizonStatic.Options.IgnoreMissingServices)
+            {
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1009)]
+        public Result LeaveRestrictedSystemSettings()
+        {
+            if (HorizonStatic.Options.IgnoreMissingServices)
+            {
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1010)]
+        public Result IsRestrictedSystemSettingsEntered(out bool arg0)
+        {
+            if (HorizonStatic.Options.IgnoreMissingServices)
+            {
+                arg0 = default;
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1011)]
+        public Result RevertRestrictedSystemSettingsEntered()
+        {
+            if (HorizonStatic.Options.IgnoreMissingServices)
+            {
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1012)]
+        public Result GetRestrictedFeatures(out int arg0)
+        {
+            if (HorizonStatic.Options.IgnoreMissingServices)
+            {
+                arg0 = default;
+                return Result.Success;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        [CmifCommand(1013)] // 4.0.0+
```

Comment:
```suggestion
        [CmifCommand(1013)]
```

We do not put firmware version as comment anymore.